examples/cpp/Makefile-2

```diff
@@ -0,0 +1,66 @@
+MOLE = ../../src/cpp
```

Comment:
Hi @arrowguy234 thank you so much for your contribution. 

Since you are adding an example in the `examples/cpp` directory, where there is already a Makefile, would you consider adding only your bit to the existing Makefile? 

Similar to what Amey did in his PR (for reference, see it [here](https://github.com/csrc-sdsu/mole/pull/27/files)). 

I think it would be cleaner and avoid duplication of code (of Makefile in this instance). Thank you!

Reply:
Now have made minor change to the existing makefile by adding my example flags , I have followed the same method that has been used for other examples.

Reply:
I @arrowguy234 , thank you for addressing the comments. However, you are still pushing a whole new file to this PR (see attached screenshot of the PR files). 
![image](https://github.com/user-attachments/assets/a5ebbc71-e728-4b6f-ad7f-35f657ae8cf1)

Instead, you want to modify the existing Makefile and push the existing Makefile to the PR. Thank you

Reply:
Sorry for the misunderstanding, i have done changes to existing makefile